engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -651,6 +647,9 @@ private void execute() throws YarnException, IOException
     int minVcores = conf.getInt("yarn.scheduler.minimum-allocation-vcores", 0);
     LOG.info("Max mem {}m, Min mem {}m, Max vcores {} and Min vcores {} capabililty of resources in this cluster ", maxMem, minMem, maxVcores, minVcores);
 
+    int maxConsecutiveContainerFailures = conf.getInt("MAX_CONSECUTIVE_CONTAINER_FAILURES", MAX_CONTAINER_FAILURES_PER_NODE);
```

Comment:
This looks like a hack. How do these configuration values make it into the conf object?